src/pages/signin/SignInPageLayout/BackgroundImage/index.android.js

```diff
@@ -1,3 +1,24 @@
-import backgroundImage from '../../../../../assets/images/home-background--android.svg';
+import React from 'react';
+import PropTypes from 'prop-types';
+import AndroidBackgroundImage from '../../../../../assets/images/home-background--android.svg';
+import styles from '../../../../styles/styles';
 
-export default backgroundImage;
+const propTypes = {
+    pointerEvents: PropTypes.string.isRequired,
```

Comment:
Please extract the propTypes to a separate file that's shared between `index.js` and `index.android.js`. Also be sure to include JSDoc comments for each prop. Thanks!